wsit/boms/bom-ext/pom.xml

```diff
@@ -36,16 +36,11 @@
         <cdi-api.version>4.0.1</cdi-api.version>
         <inject-api.version>2.0.1</inject-api.version>
         <el-api.version>5.0.0</el-api.version>
-
-        <!-- CQ: #24092 -->
-        <santuario.version>3.0.1</santuario.version>
-        <!-- CQ: #22187 -->
-        <slf4j-api.version>1.7.36</slf4j-api.version>
-        <!-- CQ: #22186 -->
+        <santuario.version>4.0.0-M1</santuario.version>
         <commons-codec.version>1.15</commons-codec.version>
 
-        <!-- CQ: #18393 -->
         <stax-utils.version>20070216</stax-utils.version>
+        <woodstox-core.version>6.4.0</woodstox-core.version>
```

Comment:
dependency on woodstox does not belong to this project; it is managed by jaxws-ri (metro-jaxws)

Reply:
Ok, rebased, and I removed this again.